### PR TITLE
Minimum Gradle Version check: 2.4

### PR DESCRIPTION
**Pull request by brunobowden**
- Throws exception for version 2.3 and earlier

### CODE REVIEW

**Comment by brunobowden** [comment]
@advayDev1, @confile - I think this is a good precautionary check to have in the plugin. I've tested it from the command line as well.

File: src/main/groovy/com/github/j2objccontrib/j2objcgradle/tasks/Utils.groovy [hunk 1]
```diff
@@ -48,6 +48,17 @@ class Utils {
     // TODO: ideally bundle j2objc binaries with plugin jar and load at runtime with
     // TODO: ClassLoader.getResourceAsStream(), extract, chmod and then execute
 
+    // Prevent construction of this class, confines usage to static methods
+    private Utils() { }
+
+    static void checkMinGradleVersion(String gradleVersion) {
```
Comment (advayDev1) [inline src/main/groovy/com/github/j2objccontrib/j2objcgradle/tasks/Utils.groovy]: TODO will fail when they get to gradle 10

**Comment by advayDev1** [comment]
lgtm.  you're introducing a y2k =)

**Comment by brunobowden** [comment]
Just for you @advayDev1, I search for the solution to our Y2K problem. There is a DefaultArtifactVersion from Apache... but I didn't want to introduce a new dependency. Then I found the GradleVersion class... which very neatly encapsulates what we need. Now version "10.0" is passing in the newly added test.

**Pull request merged**